tools/hicup/.shed.yml

```diff
@@ -0,0 +1,10 @@
+name: hicup
+owner: bgruening
+description: The HiCUP-Pipeline from the Bioinformatics Babraham Institute.
+
+long_description: A galaxy wrapper implementation of the HiCUP-Pipeline from the Bioinformatics Babraham Institute. See http://www.bioinformatics.babraham.ac.uk/projects/hicup/.
+  
```

Comment:
Can you change this to be a tool suite, rather than one repo. For example like the openms tools?